lib/eventmesh/Etcd3EventMeshServer.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const config = require('../config');
+const logger = require('../logger');
+const CONST = require('../../lib/constants');
+const EventMeshServer = require('./EventMeshServer');
+const {
+  Etcd3
+} = require('etcd3');
+
+const etcd = new Etcd3({
+  hosts: config.etcd.url[0]
+});
+
+class Etcd3EventMeshServer extends EventMeshServer {
+
+  registerService(resourceType, serviceId, serviceAttributesValue, servicePlansValue) {
+    const serviceFolderName = `services/${resourceType}/${serviceId}`;
```

Comment:
Can refactor to have function which will return serviceFolderName on passing it resoureType and serviceId instead of duplicating the same concatenation in all methods in this class ?

Reply:
Done